README.md

```diff
@@ -60,6 +60,7 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
 
 ## Security and PKI
 
+  * http://vaddy.net - Continuous web security testing with continuous integration (CI) tools
```

Comment:
do you mind adding some details, like '3 domains, 10 scan history for free' or similar